### PR TITLE
Regtest v2 deployed + pool token naming convention 

### PR DESCRIPTION
**Pull request by hyossid**

### CODE REVIEW

**Review by JBotwina — APPROVED** [review]
Looking good. Since we are directly changing the contracts on main, our dev branch needs to catch up. I fear that main and dev have diverged too far at the moment and we should work hard to sync them this week.

File: clarity/contracts/pool-token-yield-usda-59760-usda.clar [hunk 1]
```diff
@@ -0,0 +1,95 @@
+(impl-trait .trait-sip-010.sip-010-trait)
+(impl-trait .trait-pool-token.pool-token-trait)
+
+(define-fungible-token ayusda-usda)
```
Comment (fiftyeightandeight) [inline clarity/contracts/pool-token-yield-usda-59760-usda.clar:4]: the name of the fungible-token does not include all the relevant information (i.e. expiry of yield token and token)

File: clarity/contracts/pool-token-yield-usda-59760-usda.clar [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    response (begin
+      (print memo)
+      (ok response)
+    )
+    error (err error)
+  )
+)
+
+(define-read-only (get-name)
+  (ok "ayUSDA USDA LP Token")
```
Comment (fiftyeightandeight) [inline clarity/contracts/pool-token-yield-usda-59760-usda.clar:24]: See above

File: clarity/contracts/pool-token-yield-usda-59760-usda.clar [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+)
+
+(define-read-only (get-symbol)
+  (ok "AYUSDAUSDA")
```
Comment (fiftyeightandeight) [inline clarity/contracts/pool-token-yield-usda-59760-usda.clar:28]: See above

File: clarity/contracts/multisig-yield-usda-wbtc.clar [hunk 4]
```diff
@@ -130,8 +130,8 @@
     (new-fee-rate-aytoken uint)
   )
```
Comment (fiftyeightandeight) [inline clarity/contracts/multisig-yield-usda-wbtc.clar:131]: contract name should include all the relevant information, including expiry.

**Review by fiftyeightandeight — CHANGES_REQUESTED** [review]
Please review my comments and make the appropriate changes. Thank you.

**Comment by hyossid** [comment]
All fixed, please review again @fiftyeightandeight

**Pull request merged**